exchanges/cryptodotcom/cryptodotcom_test.go

```diff
@@ -778,30 +780,30 @@ func TestWsSetCancelOnDisconnect(t *testing.T) {
 
 func TestGetCreateParamMap(t *testing.T) {
 	t.Parallel()
-	arg := &CreateOrderParam{InstrumentName: "BTC_USDT", ClientOrderID: "", TimeInForce: "", Side: order.Buy, OrderType: order.Limit, PostOnly: false, TriggerPrice: 0, Price: 123, Quantity: 12, Notional: 0}
+	arg := &CreateOrderParam{InstrumentName: "BTC_USDT", Side: order.Buy, OrderType: orderTypeToString(order.Limit), Price: 123, Quantity: 12}
```

Comment:
[Previous nit not complete](https://github.com/thrasher-corp/gocryptotrader/pull/1122#discussion_r1099327224)